packages/spark-core/components/_icons.scss

```diff
@@ -13,3 +13,12 @@
 .sprk-c-Icon--large {
   height: $font-size-body-two + 0.25;
 }
+
+.sprk-c-Icon--currentColor {
```

Comment:
Should this just be the default for any icon? Like `.sprk-c-Icon` has `fill: currentColor` set and then a particular icon might override it if needed, like `.sprk-c-Icon--exclamation { fill: red }`?

Reply:
I originally had it like that but added it as a modifier because I waa seeing examples on apps where they had the icon mostly different color than the text. I’d rather have it apart of the default icon class and sounds like you would too so I’ll revert it back to the way I had originally. Woot!